Makefile

```diff
@@ -841,9 +843,6 @@ buildbox-grpc:
 		lib/multiplexer/test/ping.proto \
 		lib/web/envelope.proto
 
-# we eval within the make target to avoid invoking `go run` with every other call to the makefile
-	$(eval API_IMPORT_PATH := $(shell go run build.assets/gomod/print-import-path/main.go ./api))
```

Comment:
Can we instead change this to `$(eval API_IMPORT_PATH := $(shell head -1 api/go.mod | awk '{print $$2}'))` to catch when the api path is changed between major versions.

Reply:
Done, also pushed variables to target scope.